test/fast/Unit tests/nvm_die_on_prefix

```diff
@@ -1,11 +1,23 @@
 #!/bin/sh
 
+TEST_PWD=$(pwd)
+TEST_DIR="$TEST_PWD/nvm_die_on_prefix_tmp"
+
 cleanup () {
+  rm -rf "$TEST_DIR"
   alias nvm_has='\nvm_has'
   alias npm='\npm'
   unset -f nvm_has npm
 }
-die () { echo "$@" ; exit 1; }
+
+die () {
+  echo "$@";
+  cleanup;
+  exit 1;
+}
+
+rm -rf "$TEST_DIR"
```

Comment:
this seems like it belongs only in cleanup, not on every test run?

Reply:
Updated